src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasRelationSlicer.java

```diff
@@ -808,7 +818,7 @@ private void preProcessMultiPolygonRelation(final Relation relation)
      * @param relation
      *            The {@link Relation} to slice
      */
-    private List<TemporaryRelation> sliceRelation(final Relation relation, final List<Long> parents)
+    private List<TemporaryRelation> sliceRelation(final Relation relation)
```

Comment:
Parents were not used before here?

Reply:
Take a look at the PR description - we're removing recursive relation slicing because we're looking at them in order of lowest to highest.